Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Events.cs

```diff
@@ -24,8 +23,6 @@ private void ImageCropperThumb_KeyDown(object sender, KeyRoutedEventArgs e)
             if (e.Key == VirtualKey.Left)
             {
                 diffPos.X--;
-                var upKeyState = Window.Current.CoreWindow.GetAsyncKeyState(VirtualKey.Up);
-                var downKeyState = Window.Current.CoreWindow.GetAsyncKeyState(VirtualKey.Down);
```

Comment:
Looks like we re-call this on the next line and also do this again below, should we better optimize this or do we care since they're all switch states anyway?

If anything, we should make the lines below consistent (44+)

Reply:
I've updated the code to follow the same pattern in all branches.

---
In reply to: [390622883](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3155#discussion_r390622883) [](ancestors = 390622883)